docs/tracing.rst

```diff
@@ -236,7 +236,8 @@ following JSON structure::
         "runId": <uuid>,
         "event": <started|process_submitted|process_started|process_completed|error|completed>,
         "utcTime": <UTC timestamp>,
-        "trace": { ... }
+        "trace": { ... },
+        "manifest": { ... }
    }
```

Comment:
I think it's `metadata` instead of `manifest`, no?

Reply:
hah, nice spot